### ISSUE TITLE
"CSS minification failed" errors should provide more context as to which files are being minified

### ISSUE THREAD

**Issue by edmorley** [gruntjs/grunt-contrib-cssmin#156]
Breaking out from https://github.com/gruntjs/grunt-contrib-cssmin/issues/128, since there are both grunt-contrib-cssmin and clean-css changes required, not just clean-css.

STR:
1) Accidentally add a bogus file inside the  tags. eg:

```
<!-- build:css css/logviewer.min.css -->
<link href="css/bootstrap.css" rel="stylesheet" media="screen">
<link href="css/treeherder.css" rel="stylesheet" type="text/css">
<link href="css/logviewer.css" rel="stylesheet" type="text/css">
<link id="favicon" type="image/png" rel="shortcut icon" href="img/tree_open.png"/>
<!-- endbuild -->
```

2) Run css-min as part of a grunt build

Expected:

```
Running "cssmin:generated" (cssmin) task
File dist\css\index.min.css created: 177.13 kB → 142.92 kB
File dist\css\help.min.css created: 149.74 kB → 117.86 kB
>> RangeError: Maximum call stack size exceeded
Warning: CSS minification failed whilst generating css/logviewer.min.css. Use --force to continue.
```

Actual:

```
Running "cssmin:generated" (cssmin) task
File dist\css\index.min.css created: 177.13 kB → 142.92 kB
File dist\css\help.min.css created: 149.74 kB → 117.86 kB
>> RangeError: Maximum call stack size exceeded
Warning: CSS minification failed. Use --force to continue.
```

Note: There is https://github.com/jakubpawlowicz/clean-css/issues/428 filed to improve the clean-css error (the "RangeError: Maximum call stack size exceeded") - however the request here is specifically to add filenames somewhere, which can only be done on the grunt-contrib-cssmin side.

**Comment by edmorley** [comment]
Or alternatively, listing out the filenames of all of the files that were concatenated would perhaps be even more useful. (Just anything to give some more context as to where the error is occurring during a large grunt build script).

**Comment by filipmi** [comment]
I'd like to have a look at this issue

**Comment by edmorley** [comment]
This was fixed by #207.
